crates/util/python-utils/src/schnorr.rs

```diff
@@ -0,0 +1,62 @@
+use std::str::FromStr;
+
+use pyo3::{pyfunction, PyResult};
+use secp256k1::{schnorr::Signature, Keypair, Message, SecretKey, SECP256K1};
+use strata_crypto::{
+    sign_schnorr_sig as schnorr_sig_sign, verify_schnorr_sig as schnorr_sig_verify,
```

Comment:
Just add a `_inner` suffix instead of reordering the name, so it's more clear what's happening.